lib/Containers/FlatHashMap.h

```diff
@@ -60,4 +64,24 @@ template<class K, class V,
          class = std::enable_if<detail::MapSizeofChecker<32, 64, K, V>(), void>>
 using FlatHashMap = absl::flat_hash_map<K, V, Hash, Eq, Allocator>;
 
+namespace pmr {
+template<class K, class V,
+         class Hash = typename absl::flat_hash_map<K, V>::hasher,
+         class Eq = typename absl::flat_hash_map<K, V, Hash>::key_equal>
+using FlatHashMap = arangodb::containers::FlatHashMap<
+    K, V, Hash, Eq, std::pmr::polymorphic_allocator<std::pair<const K, V>>>;
+}
+
 }  // namespace arangodb::containers
+
+// THIS IS JUST TEMPORARY!
```

Comment:
https://github.com/abseil/abseil-cpp/blob/master/absl/hash/internal/hash.h#LL532C1-L537C2

Reply:
I have no idea what this should tell me.

Reply:
absl hash allow to makes overloads for your own values

Reply:
I saw that you write the current container_internal specialization is temporary, so I pointed how it could be done

Reply:
but how to make it transparent?

Reply:
Define your own Hash/Eq. So you don't want to write it now and just make internal overload, ok